v4/source/channels.yaml

```diff
@@ -189,6 +189,42 @@
             // GetPublicChannelsByIdsForTeam
             channels, resp := Client.GetPublicChannelsByIdsForTeam(<TEAMID>, channelIds)
 
+  /channels/{channel_id}/timezones:
+    get:
+      tags:
+        - channels
+      summary: Get timezones in a channel
+      description: |
+        Get a list of timezones for the users who are in this channel.
+        ##### Permissions
+        Must have the `read_channel` permission.
+      parameters:
+        - name: channel_id
+          in: path
+          description: Channel GUID
+          required: true
+          type: string
+      responses:
+        '200':
+          description: Timezone retrieval successful
+          schema:
+            $ref: '#/definitions/Timezone'
```

Comment:
This should be an array instead of a single object. There's examples elsewhere in this file of how to do that

Reply:
done